manifests/example/upload-datavolume.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: upload-datavolume
+spec:
+  source:
+      upload: true
+  pvc:
+    storageClassName: "hostpath"
```

Comment:
Do we really need to specify a storageClassName in the example? If someone doesn't have this particular storage class it will fail.

Reply:
yeah, I thought about too, but datavolume.yaml contains this too, so I followed this. What do you think?

Reply:
I'd say leave it in but comment it out with a note saying it's optional if you don't want to use the default storage class.

Reply:
datavolume.yaml shouldn't have it either actually, but we can fix that in another PR.